manim/mobject/svg/svg_mobject.py

```diff
@@ -28,6 +29,97 @@ def string_to_numbers(num_string):
     return [float(s) for s in re.split("[ ,]", num_string) if s != ""]
 
 
+def cascade_element_style(element, inherited):
```

Comment:
Completed in 6a057d1. I haven't done typings in Python before, so while I did Google a lot of things, it might be good to double-check what I've done.